src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/fix.module.scss

```diff
@@ -50,10 +50,12 @@
     .navArrow {
       flex: 0 0 auto;
       display: none;
+      position: absolute;
       align-items: center;
       justify-content: center;
       padding: $spacing-md;
       border-radius: $border-radius-sm;
+      right: $spacing-md;
```

Comment:
I don't think this element needs to be absolutely positioned, since it's not overlapping anything, so since absolute positioning can be a bit of a headache, I'd like to avoid it. If we want it to be located more to the right, couldn't we just reduce the inline marging of `.fixSection`?